src/init/mod.rs

```diff
@@ -229,7 +229,7 @@ pub fn init_main(shell_name: &str) -> io::Result<()> {
 
 fn print_script(script: &str, path: &str) {
     let script = script.replace("::STARSHIP::", path);
-    print!("{}", script);
+    println!("{}", script);
```

Comment:
The files typically already have a final newline.
```suggestion
    printl!("{}", script);
```

Reply:
You mean `print` right? Fixed it in my commit [06e1c62](https://github.com/starship/starship/pull/4429/commits/06e1c6296395a1e3e69e6d646ade59323372ac25)